tests/tests_integration/test_api/test_datapoints.py

```diff
@@ -58,30 +68,32 @@
 }
 DPS_TYPES = (Datapoints, DatapointsArray)
 DPS_LST_TYPES = (DatapointsList, DatapointsArrayList)
+TEST_PREFIX = "PYSDK integration test "
 
 
 @pytest.fixture(scope="session")
-def all_test_time_series(cognite_client):
-    prefix = "PYSDK integration test"
+def all_test_time_series(cognite_client) -> TimeSeriesList:
     return cognite_client.time_series.retrieve_multiple(
         external_ids=[
-            f"{prefix} 001: outside points, numeric",
-            f"{prefix} 002: outside points, string",
-            *[f"{prefix} {i:03d}: weekly values, 1950-2000, numeric" for i in range(3, 54)],
-            *[f"{prefix} {i:03d}: weekly values, 1950-2000, string" for i in range(54, 104)],
-            f"{prefix} 104: daily values, 1965-1975, numeric",
-            f"{prefix} 105: hourly values, 1969-10-01 - 1970-03-01, numeric",
-            f"{prefix} 106: every minute, 1969-12-31 - 1970-01-02, numeric",
-            f"{prefix} 107: every second, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
-            f"{prefix} 108: every millisecond, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
-            f"{prefix} 109: daily values, is_step=True, 1965-1975, numeric",
-            f"{prefix} 110: hourly values, is_step=True, 1969-10-01 - 1970-03-01, numeric",
-            f"{prefix} 111: every minute, is_step=True, 1969-12-31 - 1970-01-02, numeric",
-            f"{prefix} 112: every second, is_step=True, 1969-12-31 23:30:00 - 1970-01-01 00:30:00, numeric",
-            f"{prefix} 113: every millisecond, is_step=True, 1969-12-31 23:59:58.500 - 1970-01-01 00:00:01.500, numeric",
-            f"{prefix} 114: 1mill dps, random distribution, 1950-2020, numeric",
-            f"{prefix} 115: 1mill dps, random distribution, 1950-2020, string",
-            f"{prefix} 116: 5mill dps, 2k dps (.1s res) burst per day, 2000-01-01 12:00:00 - 2013-09-08 12:03:19.900, numeric",
+            f"{TEST_PREFIX}001: outside points, numeric",
```

Comment:
This quote came to mind 😅 

<img width="556" alt="image" src="https://user-images.githubusercontent.com/8521241/231308355-3da753d0-1bfe-4848-9a80-02ad2b386edb.png">

Reply:
...but seriously, at least keep the space before the number to improve readability

Reply:
Thanks, I try to not go to overboard. This one comes from way to many bugs caused by misspelling a string, so better to have all strings as variables (also get easy lookup of their use as a bonus).